docs/data/charts/hooks/use-scale.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Charts - useScale
+productId: x-charts
+---
+
+# useScale
+
+<p class="description">Access D3 scale functions for coordinate transformations.</p>
+
+The scale hooks provide access to D3 scale functions that can be used to convert data values to pixel coordinates within the chart.
```

Comment:
```suggestion
The scale hooks provide access to D3 scale functions that can be used to convert data values to pixel coordinates within the chart.
Scale hooks accept an axis id as a parameter.
If provided, the hook returns the scale of the associated axis.
Otherwise it returns the scale of the default axis.
```